src/realm/realmReducers.js

```diff
@@ -26,13 +28,16 @@ import {
   EVENT_REALM_FILTER_UPDATE,
 } from '../actionConstants';
 
+import { codePointMap } from '../emoji/codePointMap';
+
 // Initial state
 const initialState = {
   canCreateStreams: true,
   crossRealmBots: [],
   twentyFourHourTime: false,
   pushToken: { token: '', result: '', msg: '' },
   emoji: {},
+  emojiNameToCodePoint: codePointMap,
```

Comment:
I am confused for this name:

* emojiNameToCodePoint
* emojiNameToCodePointMap
* codePointMap
* codePointEmojiMap
* any other

In this PR I have mostly used `codePointMap`

Thoughts?

@zulip/react-native-committers

Reply:
> What's the type of it intended to be again?

It will be an object same as the current one, hard coded in the app `codePointMap`
```
{
  '+1': '1f44d',
  '-1': '1f44e',
  '0': '0030-20e3',
  '1': '0031-20e3',
.....
};
```

Reply:
We started giving these a bit more specific types.
This looks like a `{ [string]: string }`